### ISSUE TITLE
implement take-what-you-want capital

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#3908]
Reticketed from https://github.com/gratipay/gratipay.com/issues/3433#issuecomment-169544014.

> Gosh. @mitar [bringing up](https://github.com/gratipay/inside.gratipay.com/issues/458) Open Collective has me thinking about openly compensating voluntary _capital_ in addition to voluntary labor. We've been paying a lot of attention to bringing back payroll in the context of compensating labor. But what about compensating capital? We're [thinking](https://github.com/gratipay/inside.gratipay.com/issues/405) about whether capital has a role in building Gratipay itself (certainly Patreon and now Open Collective benefit from capital, seemingly at our expense). Would we need take-what-you-want pay~~roll~~_outs_ in order to bring in capital the way we want?
> 
> If we don't implement compensating voluntary _capital_ as part of this ticket, then let's be sure to reticket.
##

**Comment by whit537** [comment]
Went around the horn on this with @kaguillera here in the coffee shop. His main point is that we can solve all of the technology and legal hurdles, but the most important questions are psychological. There are very different motivations involved in investing for profit vs. giving for charity vs. working for pay. The main challenge here is to balance the three against each other.

**Comment by whit537** [comment]
We turned away venture capital in the past (https://github.com/gratipay/inside.gratipay.com/issues/68) because taking money from investors with strings attached would disincentive regular givers who were giving with no expectation of return. Teams are no longer no-strings attached, as noted at https://github.com/gratipay/inside.gratipay.com/issues/68#issuecomment-157146481. People pay on Gratipay because of a product or service that they find valuable.

What if we went a step further and made all payments investment? What if we folded investor and giver together?
